src/engine/strat_engine/backstore/backstore.rs

```diff
@@ -102,12 +102,22 @@ impl Backstore {
     /// Make a Backstore object from blockdevs that already belong to Stratis.
     /// Precondition: every device in datadevs and cachedevs has already been
     /// determined to belong to the pool with the specified pool_uuid.
+    ///
     /// Precondition: backstore_save.cap.allocs[0].length <=
     ///       the sum of the lengths of the segments allocated
     /// to the data tier cap device.
+    ///
     /// Precondition: backstore_save.data_segments is not empty. This is a
     /// consequence of the fact that metadata is saved by the pool, and if
     /// a pool exists, data has been allocated to the cap device.
+    ///
+    /// Precondition:
+    ///   * key_description.is_some() -> every StratBlockDev in datadevs has a
+    ///   key description and that key description == key_description
+    ///   *key_description.is_none() -> no StratBlockDev in datadevs has a
```

Comment:
should there be a space between * and key_description.is_none()?  If so, it is also in pool.rs

Reply:
Yes. It's meant to be a bullet. I can fix that.